src/allencell_ml_segmenter/core/file_input_widget.py

```diff
@@ -214,14 +214,18 @@ def _from_directory_slot(self) -> None:
         self._model.set_input_mode(InputMode.FROM_PATH)
 
     def _update_layer_list(self, event: Optional[NapariEvent] = None) -> None:
-        existing_selection = [
-            self._viewer.get_layers_nonthreshold()[i].name
-            for i in self._image_list.get_checked_rows()
-        ]
+        existing_selection: List[str] = []
+        for i in self._image_list.get_checked_rows():
```

Comment:
Is `FileInputWidget` the widget that displays a whole list of files in the viewer, and makes you check each one on/off ?

Reply:
I think we need a better name for the component.  the current name sounds like its just a single file select element.

Reply:
(not for this pr)